kron.py

```diff
@@ -176,16 +190,7 @@ def get_cronjobs(namespace: str = None) -> List[dict]:
         return sorted_cronjobs
 
     except ApiException as e:
-        log.error(e)
-        response = {
-            "error": 500,
-            "exception": {
-                "status": e.status,
-                "reason": e.reason,
-                "message": e.body["message"],
-            },
-        }
-        return response
+        raise _api_exception_to_http_exception(e)
```

Comment:
This passes out less information than before. Not sure if this matters since the Kubernetes CRUD model will generally give you the info you need?